scripts/reinforcement_learning/ray/tuner.py

```diff
@@ -399,6 +418,12 @@ def __init__(self, cfg: dict):
         default=MAX_LOG_EXTRACTION_ERRORS,
         help="Max number number of LogExtractionError failures before we abort the whole tuning run.",
     )
+    parser.add_argument(
+        "--progress_reporter",
+        type=str,
+        default=None,
+        help="A progress reporter in cfg_file, must be a ProgressReporter object.",
```

Comment:
The help instruction here could be improved by porividing examples of what this string may look like. If there are some fixed choices, I think it's good to mention them and use `choices` in the argument.